cloud/deployment/deployment.go

```diff
@@ -38,6 +41,9 @@ var (
 	schedulerAuMax       = 30
 	workerAuMax          = 175
 	schedulerReplicasMax = 4
+	sleepTime            = 180
+	tickNum              = 10
```

Comment:
Can you add comments here to what there variables timeouts mean? It would be easier for anyone to understand the logic